arm/ripper/makemkv.py

```diff
@@ -171,7 +171,8 @@ def process_single_tracks(job, logfile, rawpath, mode: str):
             cmd = f'makemkvcon mkv {job.config.MKV_ARGS} -r ' \
                   f'--progress={os.path.join(job.config.LOGPATH, "progress", str(job.job_id))}.log ' \
                   f'--messages=-stdout ' \
-                  f'dev:{job.devpath} {track.track_number} {shlex.quote(rawpath)}'
+                  f'dev:{job.devpath} {track.track_number} {shlex.quote(rawpath)} ' \
+                  f'--minlength={job.config.MINLENGTH}'
```

Comment:
Unfortunately I can't see the comments mentioned in the thread as I think they are still pending, however, could the solution to the manual problem here be as simple as this:

```suggestion
                  f'--minlength={job.config.MINLENGTH if mode == 'auto' else 0}'
```

This would set a minlength of 0 for any manual jobs, and all auto jobs would have the minlength set to the defined config value.